views/user/new.html

```diff
@@ -1,12 +1,24 @@
-<form action="/user/create" method="post">
-  <label for="user_email">email: </label>
-  <input type="email" name="Email" id="user_email">
-  <br/>
-  <label for="user_password">password: </label>
-  <input type="password" name="Password" id="user_password">
-  <br/>
-  <label for="user_confirm_password">confirm password: </label>
-  <input type="password" name="ConfirmPassword" id="user_confirm_password">
-  <br/>
-  <input type="submit" value="submit">
+<form class="form form--right-align" action="/user/create" method="post">
+  <fieldset>
+    <legend class="form__headline">Not signed up yet?<strong>Create a new account :)</strong></legend>
+
+    <div class="form__group">
+      <label class="form__label" for="user_email">email: </label>
+      <input class="form__input" type="email" name="email" id="user_email" placeholder="Your email address">
```

Comment:
@Lahphim How about adding `required` to each field also?

Reply:
https://github.com/Lahphim/go-crawler-challenge/pull/38/commits/06acedaae21dc7f846cd2d8a727c4f4d000788ef